tests/hil/scripts/pytest-hil/rak5010.py

```diff
@@ -0,0 +1,27 @@
+from zephyrboard import ZephyrBoard
+import subprocess
+from time import sleep
+
+class RAK5010(ZephyrBoard):
+    def __init__(self, *args, **kwargs):
+        ZephyrBoard.__init__(self, *args, **kwargs)
+
+    def program(self, fw_image):
+        command = ['gdb',
+                   '-ex', "set confirm off",
+                   '-ex', "target extended-remote {}".format(self.bmp_port),
+                   '-ex', "monitor swdp_scan",
+                   '-ex', "attach 1",
+                   '-ex', f"load {fw_image}",
+                   '-ex', "kill",
+                   '-ex', "quit"
+                   ]
```

Comment:
This is not `RAK5010` platform- specific. How about naming this class `BMPBoard` instead of `RAK5010`?